core/src/main/java/nl/nn/adapterframework/pipes/TimeoutGuardPipe.java

```diff
@@ -100,9 +100,8 @@ public PipeRunResult doPipe(Message message, IPipeLineSession session) throws Pi
 		Future future = service.submit(doPipe);
 		String result = null;
 		try {
-			log.debug(getLogPrefix(session) + "setting timeout of ["
-					+ timeout_work + "] s");
-			result = (String) future.get(timeout_work, TimeUnit.SECONDS);
+			log.debug(getLogPrefix(session) + "setting timeout of [" + timeout_work + "] s");
+			result = Message.asString(future.get(timeout_work, TimeUnit.SECONDS));
```

Comment:
Waarom asString? kan het geen message blijven?
Ik zou hier `return new PipeRunResult(getForward(), message);` verwachten

Reply:
Omdat ik er niet aan wilde zitten. We moeten van deze TimeoutGuardPipe af, omdat hij z'n werk niet afbreekt bij een timeout, maar alleen het proces door laat lopen. Het is dus nu een soort uitgestelde asynchrone excutor.
Ik zal hem deprecieren.